conf/routes

```diff
@@ -75,7 +75,7 @@ DELETE  /api/asset/:tag/attribute/:attribute   collins.app.Api.deleteAssetAttrib
 POST    /api/asset/:tag/status      collins.app.Api.updateAssetStatus(tag: String)
 PUT     /api/asset/:tag             collins.app.Api.createAsset(tag: String)
 POST    /api/asset/:tag             collins.app.Api.updateAsset(tag: String)
-DELETE  /api/asset/:tag             collins.app.Api.deleteAsset(tag: String)
+DELETE  /api/asset/:tag             collins.app.Api.deleteAsset(tag: String, nuke: Boolean)
```

Comment:
If it is in the url, it should be optional. Excluding it from the route will mean it should be read from the body, right?